frontend/src/metabase/components/UserAvatar/UserAvartar.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import UserAvatar from "./UserAvatar";
+
+export default {
+  title: "Components/UserAvatar",
+  component: UserAvatar,
+};
+
+const Template: ComponentStory<typeof UserAvatar> = args => (
+  <UserAvatar {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  user: {
+    first_name: "John",
+    last_name: "Wick",
+    email: "john.w@hightable.org",
+    common_name: "John Wick",
+  },
+};
+
+export const SingleName = Template.bind({});
+SingleName.args = {
+  user: {
+    first_name: "John",
+    last_name: null,
+    email: "john.w@hightable.org",
+    common_name: "John",
+  },
+};
+
+export const OnlyEmail = Template.bind({});
+OnlyEmail.args = {
+  user: {
+    first_name: null,
+    last_name: null,
+    email: "john.w@hightable.org",
+    common_name: "john.w@hightable.org",
+  },
+};
+
+export const shortEmail = Template.bind({});
```

Comment:
![image](https://user-images.githubusercontent.com/1937582/172845382-97780966-d5a3-4298-970b-2378dcdb9837.png)

Reply:
Dang. I missed that. Thank you!